data/ioc_extract.yaml

```diff
@@ -0,0 +1,66 @@
+# Config file for the ioc extraction analyzer.
+# A ioc extraction definition looks like this:
+# name:
+#       # IOC DB
+#       path_file_ioc:
+#       # Mandatory fields.
+#       attributes: []
+#       store_as:
+#       # Optional fields.
+#       attributes_contains: []
+#       match_re:
+#       re_flags: []
+#       tags: []
+#
+# Each definition needs to give IOC file (json format). Json sample:
+# ['ioc1', 'ioc2', 'iocx']
+# IOC DB can be make with script to collect IOC from MISP 
+# or other Threat Intel
+#
+# re_match is a regex to avoid false positive. By default is:
+# '([^a-zA-Z0-9]|^)$value$([^a-zA-Z0-9]|$)'
+# ex without re_match: '127.0.0.1' match '127.0.0.12'
+# ex with re_match by default: '127.0.0.1' not match '127.0.0.12'
```

Comment:
ok, so the re_match is a check to eliminate?

Just from reading the config file I'm not quite sure of how it works, so you provide a JSON file with exact matches, but then you've got this regular expression match, that does what exactly? For the IOC where I've got al ist of 5k domains, how does the RE affect that?

also you can just define the elastic search to be more precise here.